ui/src/app/pages/admin/admin-user-bypass.tsx

```diff
@@ -22,48 +22,61 @@ const styles = reactStyles({
   }
 });
 
-export class AdminUserBypass extends React.Component<
-    { profile: Profile},
-    { initialModules: AccessModule[], selectedModules: AccessModule[],
-      open: boolean} > {
+interface Props {
+  user: AdminTableUser;
+}
+
+interface State {
+  selectedModules: AccessModule[];
+  open: boolean;
+}
+
+const getUserModuleList = (user: AdminTableUser): Array<AccessModule> => {
```

Comment:
I'd like a name that indicates that this is the list of bypassed modules for the user - maybe `getBypassedModules()` ?